Source/UnrealGT/Public/GTRandomColorGenerator.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "CoreMinimal.h"
+
+struct UNREALGT_API GTRandomColorGenerator
```

Comment:
Structs in should be prefixed with an `F` to adhere to unreals code style.
```suggestion
struct UNREALGT_API FGTRandomColorGenerator
```

Reply:
Thanks! Still learning unreal, so all style corrections appreciated.